### PR TITLE
docs(api): update cli help usage

### PR DESCRIPTION
**Pull request by snitin315**
To be merged after https://github.com/webpack/webpack-cli/pull/2151

### CODE REVIEW

**Comment by chenxsan** [comment]
Let's merge it after a new `webpack-cli` version is out, otherwise it might confuse users.

**Comment by chenxsan** [comment]
Although a new version of `webpack-cli` is out, but there seems to be some other bugs to be fixed, so this has to be postponed again.

File: src/content/api/cli.md [hunk 1]
```diff
@@ -184,12 +184,18 @@ This is the lookup priority in increasing order
 
 W> Note that Command Line Interface has a higher precedence for the arguments you use it with than your configuration file. For instance, if you pass [`--mode="production"`](/configuration/mode/#usage) to webpack CLI and your configuration file uses `development`, `production` will be used.
 
-__List all of the commands and flags available on the cli__
+__List basic commands and flags available on the cli__
 
 ```bash
 webpack --help
```
Comment (chenxsan) [inline src/content/api/cli.md:190]:
```suggestion
npx webpack --help
```

File: src/content/api/cli.md [hunk 2]
```diff
@@ -184,12 +184,18 @@ This is the lookup priority in increasing order
...
 ```
 
+__List all supported commands and flags by cli__
+
+```bash
+webpack --help=verbose
```
Comment (chenxsan) [inline src/content/api/cli.md:196]:
```suggestion
npx webpack --help=verbose
```

**Review by chenxsan — APPROVED** [review]
I've tested it against `webpack-cli` 4.3.1, looks good.

**Comment by snitin315** [comment]
Thanks!

**Pull request merged**